workflow/run_LOH_app.cwl

```diff
@@ -3,7 +3,57 @@
 cwlVersion: v1.2
 class: Workflow
 id: run_LOH_app
```

Comment:
I would probably change to `kf-loss-of-heterozygosity-wf` to better fit others we have loaded in pub apps, or also add `preprocessing` per my review comment if deemed aproraite